ansible_collections/arista/avd/README.md

```diff
@@ -100,6 +103,9 @@ You can also open an [issue](https://github.com/aristanetworks/avd/issues) to re
 
 ## Release Notes and Roadmap
 
+<!--
+  TODO: should we add a symlink to our latest release notes to make this link less useless?
+-->
```

Comment:
Do we want to keep the link to Arista NetDevOps Examples below? the repo has not been updated in 3 years?

@carlbuchmann